spring-cloud-gcp-autoconfigure/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,7 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
+org.springframework.cloud.gcp.autoconfigure.core.GcpContextAutoConfiguration,\
+org.springframework.cloud.gcp.autoconfigure.config.GcpConfigAutoConfiguration
+
+org.springframework.cloud.bootstrap.BootstrapConfiguration=\
+org.springframework.cloud.gcp.autoconfigure.config.GcpConfigAutoConfiguration,\
+org.springframework.cloud.gcp.autoconfigure.core.GcpContextAutoConfiguration
```

Comment:
I  don't know enough about that stuff to be positive but the fact those classes are both auto-config and bootstrap config is a smell IMO.

(nothing to do with this issue per se but I thought I'd mention it).